biscuit-parser/src/parser.rs

```diff
@@ -728,6 +732,18 @@ fn name(i: &str) -> IResult<&str, &str, Error> {
     reduce(take_while1(is_name_char), " ,:(\n;")(i)
 }
 
+fn parameter_name(i: &str) -> IResult<&str, &str, Error> {
```

Comment:
this introduces a regression: 1-letter parameter names are not allowed any more

Reply:
fixed in 6d3b426